language/ruby/language.go

```diff
@@ -62,9 +66,45 @@ func (l *Language) ExecuteTests(logger *log.Logger, repositoryPath string) (test
 	return testResult, problems, nil
 }
 
+var mistakesRe = regexp.MustCompile(`\d\) Error:\n\w+#\w+:\n(.*)\n\s*(.*)`)
+
+var mistakesSyntaxErrorRe = regexp.MustCompile(`.* \(SyntaxError\)`)
```

Comment:
plz docstring, also can you name the first one more clearly? cause the second one is named more precise as well so from just looking I'm not sure what the difference is